### PR TITLE
Explicitly use TLS 1.2 in Gradle

### PR DESCRIPTION
**Pull request by breskeby**
Should fix CI failures after vault update
See https://github.com/hashicorp/vault/issues/8750\#issuecomment-631236121

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -8,3 +8,7 @@ systemProp.org.gradle.dependency.duplicate.project.detection=false
 
 # Enforce the build to fail on deprecated gradle api usage
 systemProp.org.gradle.warning.mode=fail
+
+# forcing forcing to use TLS1.2 to avoid failure in vault
```
Comment (przemekwitek) [inline gradle.properties:12]:
```suggestion
# forcing to use TLS1.2 to avoid failure in vault
```

Comment (breskeby) [reply gradle.properties]: fixed

**Review by flaper87 — APPROVED** [review]

**Comment by flaper87** [comment]
Thank y'all for working on this!

**Comment by mark-vieira** [comment]
@breskeby since vault is _only_ used in CI and via the `.ci/init.gradle` script could we configure this there instead of globally? Perhaps we can config the HTTP client used by the vault client in this way? Perhaps it's not worth it, just wondering if there are any other concerns on configuring this at the JVM level for the entire build.

**Comment by mark-vieira** [comment]
Actually this issue looks to be pretty disruptive so let's go ahead and merge as-is.

**Comment by breskeby** [comment]
Yeah I'll merge that later tonight. It doesn't make sense to differentiate between ci or not imo

**Comment by mark-vieira** [comment]
> Yeah I'll merge that later tonight. It doesn't make sense to differentiate between ci or not imo

I've already merged and backported this.

**Pull request merged**